02-Calling-an-API/src/views/ExternalApi.js

```diff
@@ -12,7 +12,7 @@ const ExternalApi = () => {
     try {
       const token = await getTokenSilently();
 
-      const response = await fetch("/api/external", {
+      const response = await fetch("http://localhost:3001/api/external", {
```

Comment:
Similar to the server-side, let's make this configurable too but default to `http://localhost:3001`.